src/JustSaying/AwsTools/MessageHandling/MessageRequester.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Amazon.SQS;
+using Amazon.SQS.Model;
+using JustSaying.Messaging.MessageProcessingStrategies;
+using JustSaying.Messaging.Monitoring;
+using Microsoft.Extensions.Logging;
+
+namespace JustSaying.AwsTools.MessageHandling
+{
+    // todo: internal?
```

Comment:
`internal` would be good as the default unless there's an obvious reason it would be useful to an external implemented to tweak the implementation. It's easy to go from `internal` to `public`, but the other way around is a breaking change.